include/git2/worktree.h

```diff
@@ -74,6 +74,14 @@ GIT_EXTERN(void) git_worktree_free(git_worktree *wt);
  */
 GIT_EXTERN(int) git_worktree_validate(const git_worktree *wt);
 
+/**
+ * Worktree add options structure
+ *
+ * Zero out for defaults.  Initialize with `GIT_WORKTREE_ADD_OPTIONS_INIT`
```

Comment:
We shouldn't recommend zeroing out the struct, but only using `GIT_WORKTREE_ADD_OPTIONS_INIT` or the corresponding function, I guess.

Reply:
IIRC this is copy-pasted from `git_checkout_options`. So what about something like :
```
/**
 * $STRUCT_DESCRIPTION
 *
 * Initialize with `$STRUCT_INIT_MACRO`. Alternatively, you can
 * use `$STRUCT_INIT_FN`.
 * 
 * @see $STRUCT_USERS
 */
```
and I take a look to make sure all conform to this style ?

Reply:
I expected that this was copied from somewhere else, but was too lazy to double-check :P Your proposal would certainly be fine with me, and removing references to zeroing out the structure are definitly welcome.

Reply:
Agreed with @pks-t, and I like this text you've provided.